emsdk_manifest.json

```diff
@@ -102,6 +102,18 @@
     "activated_cfg": "LLVM_ROOT='%installation_dir%/bin';BINARYEN_ROOT='%installation_dir%';EMSCRIPTEN_ROOT='%installation_dir%/emscripten'",
     "emscripten_releases_hash": "%releases-tag%"
   },
+  {
+    "id": "releases",
+    "version": "upstream-%releases-tag%",
+    "bitness": 64,
+    "arch": "aarch64",
+    "macos_url": "https://storage.googleapis.com/webassembly/emscripten-releases-builds/mac/%releases-tag%/wasm-binaries-arm64.tbz2",
```

Comment:
Is there a contextual difference between `aarch64` and `arm64`? I see one in the URL and one in the arch field.

Best answer I found was this: https://stackoverflow.com/a/47274698/1176841

Reply:
To my understanding it should be the same thing, but python reports the field using different strings on linux and mac M1 it looks like.

Reply:
Yes, they are the same thing, and that's a pretty good summary of some reaasons the nomenclature varies. It was pretty amusing  in LLVM during the time that ARM was developing their aarch64 backend upstream, while meanwhile it was the industry's worst-kept secret that apple was developing their own arm hardware and likely had their own private backend. Eventually they did merge the backends using some code and approaches from both (they also poached the engineer from ARM that led the upstream implementation; I'm sure that helped 😁 )
Anyway, in this patch I just kept aarch64 in the manifest since it was already there but most of the filenames have arm64 in them.

Reply:
Why not use aarch64 in the filenames too.. given that we control them?

Reply:
arm64 is more consistent with the binaries we don't control, e.g. python and node; and with other tools on mac (e.g. `uname` and Python's `platform.machine()`

Reply:
Annoying that we have to live with two names :(   oh well..